src/components/nav-drop-down.jsx

```diff
@@ -37,7 +37,7 @@ function NavDropDown() {
       const navDropDownLinks = dropDownLinks[i];
       const navDropDownLinksLength = navDropDownLinks.length;
       const navDropDownLinkLabels = dropDownLabels[i].linkLabels;
-      const navDropDownLinkLabelsLength = navDropDownLinkLabels.length;
+      const navDropDownLinkLabelsLength = navDropDownLinkLabels?.length;
```

Comment:
The logic in this file has also changed through my PRs. I would recommend merging dev into this branch and making sure what you have here works.